### PR TITLE
refactor: remove minor nolint

### PR DESCRIPTION
**Pull request by grace0925**
Signed-off-by: Grace Liu <grace.liu@securekey.com>

### CODE REVIEW

File: pkg/restapi/governance/operation/operations_test.go [hunk 1]
```diff
@@ -427,7 +427,8 @@ func serveHTTPMux(t *testing.T, handler Handler, reqBytes []byte,
 	return rr
 }
 
-func serveHTTP(t *testing.T, handler http.HandlerFunc, method, path string, req []byte) *httptest.ResponseRecorder { // nolint: unparam,lll
+//nolint:unparam
```
Comment (rolsonquadras) [inline pkg/restapi/governance/operation/operations_test.go:430]: Can we use `_` for unused param and remove this lint ?

**Review by rolsonquadras — APPROVED** [review]

File: pkg/restapi/governance/operation/operations_test.go (hunk 1 above)
Comment (grace0925) [reply pkg/restapi/governance/operation/operations_test.go:430]: there is no unused param here, it complains about "method" always being "POST"

Comment (rolsonquadras) [reply pkg/restapi/governance/operation/operations_test.go:430]: Oh ok.

Comment (grace0925) [reply pkg/restapi/governance/operation/operations_test.go:430]: so either keep it or remove 'method' and hardcode "POST"

Comment (rolsonquadras) [reply pkg/restapi/governance/operation/operations_test.go:430]: I guess, we can keep it as its a test file.

**Review by fqutishat — APPROVED** [review]

**Pull request merged**